service/history/execution/state_builder.go

```diff
@@ -97,15 +97,15 @@ func (b *stateBuilderImpl) ApplyEvents(
 	lastEvent := history[len(history)-1]
 	var newRunMutableStateBuilder MutableState
 
-	b.logger.Debugf("Applying events for domain %s, wfID %v, first event [id:%v, version:%v], last event [id:%v, version:%v]",
+	b.logger.Debugf("stateBuilderImpl Applying events for domain %s, wfID %v, first event [id:%v, version:%v], last event [id:%v, version:%v]",
 		domainID, workflowExecution.WorkflowID, firstEvent.ID, firstEvent.Version, lastEvent.ID, lastEvent.Version)
 
 	// need to clear the stickiness since workflow turned to passive
 	b.mutableState.ClearStickyness()
 
 	historyLength := len(history)
 	for i, event := range history {
-		b.logger.Debugf("Applying event %v of %v. Calling UpdateCurrentVersion for domain %s, wfID %v, event [id:%v, version:%v]",
+		b.logger.Debugf("stateBuilderImpl Applying event %v of %v. Calling UpdateCurrentVersion for domain %s, wfID %v, event [id:%v, version:%v]",
```

Comment:
this will be hard to query, I'd suggest putting these as tags where possible

Reply:
Not meant for grouping at scale but agree we should keep tagging where applicable.